pages/common/nc.md

```diff
@@ -14,7 +14,7 @@
 
 `nc -w {{timeout_in_seconds}} {{ipaddress}} {{port}}`
 
-- Serve a file:
```

Comment:
I think this is good as it is. The command `nc -l port < file` is not actually sending a file automatically. It is serving it waiting for someone to connect to the specified port.

Reply:
Yeah, let's not change this.

Reply:
Reverted